src/utils.rs

```diff
@@ -102,7 +102,7 @@ mod tests {
 
     #[test]
     fn exec_with_output_stdout() {
-        let result = internal_exec_cmd("/bin/echo", &["-n", "hello"]);
+        let result = internal_exec_cmd("/bin/sh", &["-c", "echo -n hello"]);
```

Comment:
This is failing on OSX looks the `-n` argument is getting ignored. If you change this

```suggestion
        let result = internal_exec_cmd("/bin/sh", &["-c", "echo hello"]);
```